dll/win32/getuname/lang/ro-RO.rc

```diff
@@ -24,7 +24,7 @@ BEGIN
     12 "Pagină nouă - FF"
     13 "Retur caracter purtător - CR"
     14 "Deplasament Ext - SO"
-    15 "Deplasament Int - SI"
+    15 "Deplasament int - SI"
```

Comment:
Considering the line above this, you may want to either have this word back to being capitalized, or to lower the case for "Ext" in the line above, for consistency.

Reply:
This got fixed while in review.